### PR TITLE
Adding Igbo monolingual dataset

### PR DESCRIPTION
**Pull request by purvimisal**
This PR adds the Igbo Monolingual dataset.
Data: https://github.com/IgnatiusEzeani/IGBONLP/tree/master/ig_monoling
Paper: https://arxiv.org/abs/2004.00648

### CODE REVIEW

File: datasets/igbo_monolingual/README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+## Dataset Description
+
+- **Homepage:** https://github.com/IgnatiusEzeani/IGBONLP/tree/master/ig_monoling
+- **Repository:** https://github.com/IgnatiusEzeani/IGBONLP/tree/master/ig_monoling
+- **Paper:** https://arxiv.org/abs/2004.00648
+
+### Dataset Summary
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/igbo_monolingual/README.md:72]: Can you add a summary please ?

File: datasets/igbo_monolingual/README.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/igbo_monolingual/README.md:90]: can you briefly describe the data fields depending on the config ? (content, tags, description etc.)

and also mention that missing `date` values are replaced with empty strings ?

File: datasets/igbo_monolingual/README.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+### Data Splits
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/igbo_monolingual/README.md:94]: Can you add a table with the sizes of each configuration ?

File: datasets/igbo_monolingual/README.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/igbo_monolingual/README.md:80]:
```suggestion
Igbo (ig)
```

**Review by lhoestq — COMMENTED** [review]
Really cool thank you !

I left a few comments

Also it looks like the dummy_data.zip files are quite big (500KB each). Could you try to reduce their sizes please ? Ideally they should be less than 20KB each.

To do so feel free to take a look inside the dummy_data.zip file for the bbc-igbo config for example, and remove all the unnecessary files from the other configs like igbo-radio.
Moreover for the remaining file feel free to remove big chunks of text to only keep a few passages.

**Comment by purvimisal** [comment]
@lhoestq Thank you for the review. I have made all the changes you mentioned. PTAL!

File: datasets/igbo_monolingual/README.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+  - title, content, format, date (Missing date values replaced with empty strings)
+
+For config 'jw-ulo_nche_naamu':
+  - title, content, format, date (Missing date values replaced with empty strings)
+
+
+### Data Splits
+| bbc-igbo  | eze_goes_to_school |igbo-radio| jw-books|jw-nt-igbo| jw-ot-igbo | jw-teta |jw-ulo_nche |jw-ulo_nche_naamu
+| ------------- |:-------------:|:-------------:|:-------------:|:-------------:|:-------------:|:-------------:|:-------------:|:-------------:|
+| 1K<n<10K      |n<1K     |n<1K    |n<1K     |n<1K     |n<1K|n<1K|n<1K|n<1K
```
Comment (lhoestq) [inline datasets/igbo_monolingual/README.md:121]:
```suggestion
|  1297     | 1     | 440   | 48     | 27     | 39 | 37 | 55 | 88
```

File: datasets/igbo_monolingual/README.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+### Languages
+
+Igbo (ig)
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/igbo_monolingual/README.md:86]:
````suggestion
Here is an example from the bb-igbo config:
```
{'content': 'Ike Ekweremmadụ\n\nIke ịda jụụ otụ nkeji banyere oke ogbugbu na-eme n\'ala Naijiria agwụla Ekweremmadụ\n\nOsote onye-isi ndị ome-iwu Naịjirịa bụ Ike Ekweremadu ekwuola na ike agwụla ndị Sịnatị iji otu nkeji darajụụ akwanyere ndị egburu n\'ime oke ọgbaghara dị na Naịjirịa oge ọ bula.\n\nEkweremadu  katọrọ mwakpọ na ogbugbu ndị Naịjirịa aka ha dị ọcha nke ndị Fulani na-achị ehi mere, kwuo na ike agwụla ndị ome- iwu ịkwanyere ha ugwu n\'otu nkeji\'\n\nCheta n\'otu ịzụka gara-aga ka emere akwam ozu mmadụ ruru iri asaa egburu na Local Gọọmenti Logo na Guma nke Benue Steeti, e be ihe kariri mmadụ iri ise ka akụkọ kwuru n\'egburu na Taraba Steeti.\n\nEkweremadu gosiri iwe gbasara ogbugbu ndị mmadụ na nzukọ ndị ome-iwu n\'ụbọchị taa, kwuo na Naịjirịa ga-ebu ụzọ nwe udo na nchekwa, tupu e kwuowa okwu iwulite obodo.\n\nỌ sịrị:  "Ndị ome-iwu abụghị sọ ọsọ ndị ihe a metụtara, kama ndị Naịjirịa niile.\n\n\'Ike agwụla anyị iji otu nkeji dị jụụ maka nkwanye ugwu. Ihe anyị chọrọ bụ udo na nchekwa tupu echewa echịchị nwuli obodo."',
 'date': '2018-01-19T17:07:38Z',
 'description': "N'ihi oke ogbugbu ndị mmadụ na Naịjirịa gbagburu gburu, osota onyeisi ndị ome-iwu Naịjirịa bụ Ike Ekweremadu ekwuola na ihe Naịjiria chọrọ bụ nchekwa tara ọchịchị, tupu ekwuwa okwu ihe ọzọ.",
 'headline': 'Ekweremadu: Ike agwụla ndị ụlọ ome iwu',
 'source': 'https://www.bbc.com/igbo/42712250',
 'tags': [],
 'title': 'Ekweremadu: Ike agwụla ndị ụlọ ome iwu'}
```
````

**Review by lhoestq — APPROVED** [review]
Thanks !

**Pull request merged**